SparkPost/matchbox
# UX-752 Generate meta token file

### What Changed
- Adds a custom formatter to generate the `meta` javascript file used for documentation
- Exports javascript files as:

```js
tokens,   // legacy tokens
tokens_legacy: tokens,  // legacy tokens
tokens_next,  // new tokens
meta,  // meta data
```

### How To Test or Verify
- Run `npm run build` inside the `design-tokens-sd` package
- Verify the meta file is generated correctly

### PR Checklist
